ming_sim/db.py

```
@@ -10402,6 +10402,86 @@ def _normalize_directive_dossier_payload(
+                grant_did = strict_int(
+                    normalized.get("grant_dossier_id"), accept_numeric_strings=True,
+                )
+            except (TypeError, ValueError):
+                grant_did = 0
+            if grant_did > 0:
+                normalized["grant_dossier_id"] = grant_did
+            # 身份真源在 authority_records；禁保留并行 authorization_id(s)
+            normalized.pop("authorization_id", None)
+            normalized.pop("authorization_ids", None)
+        if action == "revoke_decree":
+            # #523：目标仅承诺/旨意；纯授权不得入撤回成命。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)